client/snapper.cc

```diff
@@ -1522,7 +1523,11 @@ main(int argc, char** argv)
     catch (const runtime_error& e)
     {
 	cerr << "Failed to set locale. Fix your system." << endl;
-	exit(EXIT_FAILURE);
+    }
+
+    if (strcmp(nl_langinfo(CODESET), "UTF-8") != 0)
+    {
+	cerr << "Running in non UTF-8 locale. Setup is unsupported." << endl;
```

Comment:
Q: The STDERR messages are not translated in snapper? E.g. the `--help` output few lines above is translated, but that goes to STDOUT...

Reply:
Actually this STDERR https://github.com/openSUSE/snapper/pull/403/files#diff-b5c274f368d2fab68ee1bc4f9206df4eR1490 is translated, so I guess these two should be as well...

Reply:
AFAIS the first will never be translated since setting the locale failed. Anyway, marked to be translated.